dataikuapi/dss/dataset.py

```diff
@@ -859,3 +870,78 @@ def already_exists(self):
             return True
         except Exception as e:
             return False
+
+
+class DSSDatasetInfo(object):
+    """
+    Info class for a DSS dataset (Read-Only).
+    Do not instantiate this class directly, use :meth:`DSSDataset.get_info`
+    """
+
+    def __init__(self, dataset, info):
+        """
+        :type dataset: :class:`DSSDataset`
+        :type info: dict
+        """
+        self.dataset = dataset
+        self.info = info
+
+    def get_raw(self):
+        """
+        Get the raw dataset full information as a dict
+
+        :return: the raw dataset full information
+        :rtype: dict
+        """
+        return self.info
+
+    def get_raw_last_build(self):
+        """
+        Get the last build information of the dataset, as a raw dict, or None if there is no last build information.
+
+        :return: the last build information
+        :rtype: dict or None
+        """
+        return self.info.get("lastBuild", None)
+
+    def get_last_build_start_time(self, as_date=False):
+        """
+        Get the last build start time of the dataset as a timestamp or as a :class:`datetime.datetime`, or None if there
+        is no last build information.
+
+        :return: the last build start time
+        :rtype: int or :class:`datetime.datetime` or None
+        """
+        last_build_info = self.info.get("lastBuild", dict())
+        timestamp = last_build_info.get("buildStartTime", None)
+
+        if as_date and timestamp is not None:
+            return datetime.datetime.fromtimestamp(timestamp / 1000)
+        return timestamp
+
+    def get_last_build_end_time(self, as_date=False):
+        """
+        Get the last build end time of the dataset as a timestamp or as a :class:`datetime.datetime`, or None if there
+        is no last build information.
+
+        :return: the last build end time
+        :rtype: int or :class:`datetime.datetime` or None
+        """
+        last_build_info = self.info.get("lastBuild", dict())
+        timestamp = last_build_info.get("buildEndTime", None)
+
+        if as_date and timestamp:
+            return datetime.datetime.fromtimestamp(timestamp / 1000)
+        return timestamp
+
+    def is_last_build_successful(self):
```

Comment:
make it a property. Check what we do in scenario.py (462/466/...)

Reply:
done 👍🏻